docs/concepts/CLQL.md

```diff
@@ -296,6 +299,8 @@ class(depth = any):
 
 The query above will only return methods of `classA` for which `classB` has a corresponding method.
 
```

Comment:
Are you going to explain this query in a later PR? 

"My mental model up until this point was that "as" puts the property into a variable, but this also implies that it acts like a constraint."

Reply:
Yes these are only a handful of the changes to be made. Broke into multiple PR's to make it easier to review